tensorboard/plugins/projector/vz_projector/vz-projector-metadata-card.html

```diff
@@ -69,7 +69,7 @@
 }
     
 .metadata-value {
-  overflow-wrap: break-word;
+  word-break: break-word;
```

Comment:
I am rather appalled at how difficult styling a table is. I understand why but it is crazy. 
Instead of wasting more time, I am okay with going forward with this change with minor change.

Note that this does not work in Firefox. Funnily, `overflow-wrap: break-word` does not work either. Perhaps we want:
```css
word-wrap: anywhere; /* Firefox only -- word-wrap DNE in Chrome. anywhere DNE in Chrome */
word-break: break-word; /* break-word DNE in Firefox */
```